src/ui/progress.js

```diff
@@ -90,36 +90,40 @@ class CanvasRenderer extends BaseRenderer {
   }
 
   update() {
-    var arc, ctx, half, size
-    half = Math.floor(Math.min(this.element.width(), this.element.height()))
-    size = half * 2
-    if (half) {
-      if (this.canvasEl.width !== size || this.canvasEl.height !== size) {
-        this.canvasEl.width = size
-        this.canvasEl.height = size
+    this.__rafId = window.requestAnimationFrame(() => {
+      window.cancelAnimationFrame(this.__rafId)
```

Comment:
Why we pause animation after the start? 
It looks like:
`setTimeout(() => {}, 0)`

Reply:
Oops, we need to cancel previous frame, so cancelling should be outside raf, fixed it.

The reason is that we need to prevent frame stacking when browser pauses rafs (tab is become background for example). There is no need to call every raf callback, the last one required only.